homeassistant/components/camera/webrtc.py

```diff
@@ -342,9 +342,16 @@ async def ws_candidate(
     connection: websocket_api.ActiveConnection, msg: dict[str, Any], camera: Camera
 ) -> None:
     """Handle WebRTC candidate websocket command."""
-    await camera.async_on_webrtc_candidate(
-        msg["session_id"], RTCIceCandidateInit(msg["candidate"])
-    )
+    try:
+        candidate_init = RTCIceCandidateInit.from_dict(msg["candidate"])
+    except Exception as ex:  # noqa: BLE001
```

Comment:
Catching bare metal Exception is not allowed. Please catch specific execeptions

Reply:
Fixed